### PR TITLE
Cardio Page has been added

### PR DESCRIPTION
**Pull request by Anubhavgupta14**
Cardio Page has been added.
Kindly Review it. @TusharKesarwani @Subhradeep10 
If you like the UI of the page then I will also improve the UI of all other similar page so that all looks similar.

### CODE REVIEW

**Comment by Anubhavgupta14** [comment]
Issue No. #273

File: index.html [hunk 1]
```diff
@@ -146,11 +146,16 @@ <h2 class="title">Bicep Workout</h2>
...
+        <a href="./cardio.html">
         <div class="img">
           <img src="./images/cardio_day.jpg" alt="" />
         </div>
+      </a>
         <div class="content">
-          <h2 class="title">Cardio Workout</h2>
+          <a href="./cardio.html">
+            <h2 class="title">Cardio Workout</h2>
+          </div>
```
Comment (TusharKesarwani) [inline index.html:157]: Why you are closing div in this section

Comment (TusharKesarwani) [reply index.html:157]: Remove it and push it again

Comment (Anubhavgupta14) [reply index.html]: Now, it has been resolved. @TusharKesarwani

Comment (Anubhavgupta14) [reply index.html]: It was a typing error.

**Review by TusharKesarwani — APPROVED** [review]

**Comment by Anubhavgupta14** [comment]
@Subhradeep10 Kindly Review it and if its ok then I request you to merge it.

**Pull request merged**